runtime-light/streams/streams.h

```diff
@@ -16,6 +16,8 @@ enum class DescriptorRuntimeStatus {
   Timer
 };
 
+constexpr uint64_t BAD_PLATFORM_DESCRIPTOR = 0;
```

Comment:
Are you sure that `0` is "bad" descriptor in the platform?

Reply:
Yes